Analytics/IAnalytics.h

```diff
@@ -0,0 +1,60 @@
+/**
```

Comment:
We shouldn't add the interface header file here directly. It should reside in ThunderInterfaces repo.